service/application/src/test/java/com/solace/maas/ep/event/management/agent/testConfigs/PublisherTestConfig.java

```diff
@@ -19,11 +18,7 @@
 @Profile("TEST")
 @Slf4j
 public class PublisherTestConfig {
-    @Bean
-    @Primary
-    public MessagingService messagingService() {
-        return mock(MessagingService.class);
-    }
```

Comment:
This is moved to https://github.com/SolaceProducts/event-management-agent/pull/182/files#r1630520276 with more detailed behaviour mock